modules/core/src/system.cpp

```diff
@@ -1370,6 +1370,8 @@ bool __termination = false;
 
 //////////////////////////////// thread-local storage ////////////////////////////////
 
+#include <atomic>
```

Comment:
This include is within the cv namespace and that creates a compiler error here. Maybe just move it to the top ?